### ISSUE TITLE
NCBI DB  needs more information in the ID

### ISSUE THREAD

**Issue by andersgs** [tseemann/abricate#74]
In the NCBI AMR Gene Reference Browser (https://www.ncbi.nlm.nih.gov/pathogens/isolates#/refgene/), locus_tag is not a possible field. It would be great if the sequence IDs included beyond the locus_tag the protein iD or refseq nucleotide accession (with version), so that a table of results can be easily joined with the ref gene table.

**Comment by tseemann** [comment]
This is what i get from a gene:

```
LOCUS       NG_060581               1783 bp    DNA     linear   CON 27-JUN-2019
DEFINITION  Klebsiella pneumoniae subsp. pneumoniae KPCTRSRTH02 mcr-3 gene for
            phosphoethanolamine--lipid A transferase MCR-3.22, complete CDS.
ACCESSION   NG_060581
<snip>
FEATURES             Location/Qualifiers
     source          1..1783
                     /organism="Klebsiella pneumoniae subsp. pneumoniae"
                     /mol_type="genomic DNA"
                     /strain="KPCTRSRTH02"
                     /isolation_source="Rectal swab"
                     /host="Swine"
                     /sub_species="pneumoniae"
                     /db_xref="taxon:72407"
                     /country="Thailand: Chachoengsao"
                     /lat_lon="13.6904 N 101.0780 E"
                     /collection_date="2016-08-11"
     gene            58..1683
                     /gene="mcr-3"
                     /locus_tag="A7J11_05800"
                     /allele="mcr-3.22"
     CDS             58..1683
                     /gene="mcr-3"
                     /locus_tag="A7J11_05800"
                     /allele="mcr-3.22"
                     /codon_start=1
                     /transl_table=11
                     /product="phosphoethanolamine--lipid A transferase
                     MCR-3.22"
                     /protein_id="WP_094308975.1"
<snip>
ORIGIN
        1 attttttaca aagatgtctc gttagaaagt gattgttgga ctattaatgg agtaagtatg
       61 ccttccctta taaaaataaa aattgttccg cttatgttct ttttggcact gtattttgca
      121 tttatgctga actggcgtgg agttctccat ttttacgaaa tcctttacaa attagaagat
      181 tttaagtttg gtttcgccat ttcattacca atattgcttg ttgcagcgct taactttgta
<snip>
//
```

**Comment by tseemann** [comment]
THis has been fixed by #101 - the ACCESSION is now the `NG_060581.1` style ID.
